src/viewport_func.h

```diff
@@ -93,4 +93,10 @@ static inline void MarkTileDirtyByTile(TileIndex tile, int bridge_level_offset =
 
 Point GetViewportStationMiddle(const ViewPort *vp, const Station *st);
 
+struct Station;
+struct Town;
+
+void ViewportCatchmentStation(const Station *st, bool sel);
+void ViewportCatchmentTown(const Town *t, bool sel);
```

Comment:
I think these need a Set or Show in the function name. Without a verb they sound like type names.

Reply:
Also done